pkg/revisiongenerator/revision_test.go

```
@@ -530,6 +578,142 @@ data:
+		g.Expect(compatObjects[0].GetName()).To(Equal("ccapio-widgets.example.com"))
+		g.Expect(compatObjects[1].GetName()).To(Equal("ccapio-gadgets.example.com"))
+
+		g.Expect(components[1].CRDs()).To(BeEmpty(), "core CRDs should be empty after removal")
+		g.Expect(components[2].CRDs()).To(BeEmpty(), "infra CRDs should be empty after removal")
+	})
+
+	t.Run("object collectors invoked for CompatibilityRequirement objects", func(t *testing.T) {
+		g := NewWithT(t)
+
+		var collected []string
+		collector := func(obj unstructured.Unstructured) {
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)